.gitignore

```diff
@@ -23,3 +23,6 @@ cabal.project.local
 cabal.project.local~
 .HTF/
 .ghc.environment.*
+
+.envrc
+.direnv/
```

Comment:
These are files that aren't generated by the project, but by the user themselves. I'm a proponent of letting the user add these in their own `~/.config/git/ignore` instead of here if they're using direnv.